did-pkh-method-draft.md

```diff
@@ -1,43 +1,56 @@
 # did:pkh Method Specification
+
 Authors: Wayne Chang, Charles Lehner, Juan Caballero, Joel Thorstensson
 Status: Draft
+
 ## Introduction
+
 ### Problem Statement
+
 There are over hundreds of billions of on-chain, balance-holding accounts across
 the major 50 or so blockchains, all secured and namespaced using similar
 technologies. Almost all of these derive identifiers from hashed or otherwise
 obscured public keys, which are provided at time of transaction.
+
 These accounts are used to protect billions of dollars in assets and critical
 digital infrastructure. They are also actively being piloted and used by
 enterprises and governments. They are rapidly becoming a major form of shared
 data infrastructure across verticals and continents.
+
 DIDs should favor usability where possible, and it is extremely beneficial from
 a security & human computer interaction perspective to have DIDs that readily
 correspond to their equivalents on decentralized networks. This corresponds
 neatly to end-users' understanding of what an "account" is on an existing
 network.  There are knock-on security effects to having an
 immediately-recognizable "address" double as a DID for usage elsewhere.  
+
 It also allows most if not all blockchain accounts to instantly leverage an
 existing identity/account and deploy a W3C Decentralized Identifier from it in a
 standards-conformant way. This "DID-wrapping" of an existing identifier can be
 used in combination with other DID-compatible technologies, such as W3C
 Verifiable Credentials or Authorization Capabilities, and produce proper
 signature-suite definitions, such as "metamask-signing" (signing according to
 the [[eip712]] protocol, soon to be a work item at W3C-CCG).
+
 ### Relationship to other DID architectures
+
 did:pkh bears many similarities to
 [did:key](https://w3c-ccg.github.io/did-method-key/#introduction) except it is
 optimized for identifiers derived from hashes of public keys according to
 well-known algorithms (commonly referred to as "public key hashes" because in
 most cases they are a public key hashed according to a standard hash function).
```

Comment:
```suggestion
`did:pkh` is similar in many ways to
[`did:key`](https://w3c-ccg.github.io/did-method-key/#introduction), except that
`did:pkh` is optimized for identifiers derived from hashes of public keys according
to well-known algorithms (commonly referred to as "public key hashes", because, in
most cases, they are a public key hashed according to a standard hash function).
```